inhouse/command_handlers/soloqueue_leaderboard.py

```diff
@@ -13,6 +18,22 @@ def __init__(self) -> None:
         self.bronze_dict = {"I":[],"II":[],"III":[],"IV":[]}
         self.iron_dict = {"I":[],"II":[],"III":[],"IV":[]}
         self.unranked_dict = []
```

Comment:
I think all of these in the initializer can just be replaced with a `self.clearDicts()` for cleanliness.